app/modules/intelligence/tools/code_query_tools/get_file_content_by_path.py

```
@@ -214,11 +432,22 @@ def _run(
                 "content": content,
             }
         except FileNotFoundError as e:
```

Ruff (0.15.1)

[error] Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)